misk/src/main/kotlin/misk/web/interceptors/ResponseBodyMarshallerFactory.kt

```diff
@@ -0,0 +1,40 @@
+package misk.web.interceptors
```

Comment:
copyright

Reply:
I see some existing files don't have copyright either, we don't care?

Reply:
We have it in the root of the project!